index.bs

```diff
@@ -806,6 +806,12 @@ An attribution source is a [=struct=] with the following items:
 :: A [=trigger-data matching mode=].
 : <dfn>debug cookie set</dfn> (default false)
 :: A [=boolean=].
+: <dfn>attribution scope limit</dfn>
+:: Null or a non-negative 32-bit integer, number of distinct [=attribution source/attribution scopes=] allowed per [=attribution source/reporting origin=].
+: <dfn>attribution scopes</dfn>
+:: Null or a list of non-negative 32-bit integers.
+: <dfn>max number of view states</dfn>
+:: Null or a non-negative 32-bit integer, maximum number view states that an API caller plans to use.
```

Comment:
Please clarify what a "view state" is. That term doesn't appear elsewhere in the specification.